katello/hooks/post/31-cdn_setting.rb

```diff
@@ -0,0 +1,10 @@
+if module_enabled?('katello')
```

Comment:
This should respect `noop`
```suggestion
if !app_value(:noop) && module_enabled?('katello')
```